ui-modules/blueprint-composer/app/components/util/d3-blueprint.js

```diff
@@ -831,26 +832,45 @@ export function D3Blueprint(container, options) {
 
         let relationArcs = _relationArcs.selectAll('.relation').data(arcsData, (d) => getPathId(d));
 
+        // Returns node coordinates with adjustment for member spec rotation if present.
+        const getNodeCoordinates = (entity) => {
+            let _node = nodeForEntity(entity);
+            let _x = _node.x;
+            let _y = _node.y;
+
+            if (entity.isMemberSpec()) {
+                _y += _configHolder.nodes.memberspec.circle.cy;
```

Comment:
This is what was happening before, member spec goes down on _Y_  axis by `_configHolder.nodes.memberspec.circle.cy` amount, and this becomes a radius for rotation when parent node gets a children.